chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts

```diff
@@ -1,19 +1,47 @@
 import {Component, OnInit} from '@angular/core';
-
+import {FormValidationService} from '../../services/form-validation.service/form-validation.service';
+import {MemberService} from '../../services/member.service/member.service';
 @Component({
   selector: 'app-sign-up',
   templateUrl: './sign-up.component.html',
   styleUrls: ['./sign-up.component.css']
 })
 export class SignUpComponent implements OnInit {
+  // Todo: create an errorMessage attribute so that it can be use later if and when the memberService.createAccount() return false.
+ public setErrorMessage(querySelector, errorMessage, input) {
+    document.querySelector(querySelector).innerHTML = errorMessage;
+    input.setCustomValidity(errorMessage);
+
```

Comment:
Remove empty space.  Look in other files for empty spaces and remove them too.

Reply:
Looked for empty space and formatted code